zerver/tornado/descriptors.py

```diff
@@ -1,19 +1,15 @@
 
 from typing import Any, Dict, Optional
 
-if False:
-    import zerver.tornado.event_queue
+# from zerver.tornado.event_queue import ClientDescriptor
 
-descriptors_by_handler_id = {}  # type: Dict[int, zerver.tornado.event_queue.ClientDescriptor]
+descriptors_by_handler_id = {}  # type: Dict[int, 'ClientDescriptor']
 
-def get_descriptor_by_handler_id(handler_id):
```

Comment:
@timabbott @gnprice any idea why the class can't be imported? I face another similar issue here while manual fixing the zerver/tornado migration: https://travis-ci.org/rht/zulip/jobs/293431727#L1526

Reply:
The current stopgap solution is to just use quote.

Reply:
Oh the reason for https://travis-ci.org/rht/zulip/jobs/293431727#L1526 is that the class is defined in the same file, but later. So using quote is correct in this case for forward class definition.

Reply:
For this one, it's because we have an import loop.